tests/haskell.vader

```diff
@@ -0,0 +1,39 @@
+Include: _setup.vader
```

Comment:
Yeah, seems to be an issue with Vader (https://github.com/junegunn/vader.vim/issues/88).
Can you move it to `ft_haskell.vader` then, please?